src/Compilers/CSharp/Portable/Symbols/Attributes/AttributeData.cs

```diff
@@ -41,10 +41,6 @@ internal abstract partial class CSharpAttributeData : AttributeData
         /// </summary>
         public new abstract SyntaxReference? ApplicationSyntaxReference { get; }
 
-        // Overridden to be able to apply MemberNotNull to the new members
-        [MemberNotNullWhen(true, nameof(AttributeClass), nameof(AttributeConstructor))]
-        internal abstract override bool HasErrors { get; }
```

Comment:
>internal abstract override bool HasErrors { get; } [](http://example.com/codeflow?start=8&length=50)

We should keep an abstract override of this API because the one in base is not abstract. There is a benefit in this beyond the ability to apply an attribute.